roles/foreman_installer/tasks/main.yml

```diff
@@ -9,6 +9,9 @@
 - debug:
     msg: "Foreman installer version {{ foreman_installer_version_file['content'] | b64decode }}"
 
+- set_fact:
```

Comment:
Perhaps this should be done before the debug. The debug can then use this variable.